### ISSUE TITLE
Logback files should not be included in libraries

### ISSUE THREAD

**Issue by danielflower** [neuroph/neuroph#53]
When adding neuroph as a maven dependency, Logback dependencies and the logback.xml config file are included. This can cause, as happened to me, the neuroph logback.xml to overwrite my own logback.xml

For libraries, you should only include slf4j as a dependency.

Nice library, btw!

**Comment by Tires** [comment]
Agree, can you please remove that file? This is messing up the whole logging when you use logback as logging.
